ui/views.py

```diff
@@ -240,10 +249,6 @@ class HelpPageView(TemplateView):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         js_settings = default_js_settings(self.request)
-        if not self.request.user.is_anonymous:
-            has_admin = Collection.objects.all_admin(self.request.user).exists()
-            is_staff_or_superuser = ui_permissions.is_staff_or_superuser(self.request.user)
-            js_settings["editable"] = has_admin or is_staff_or_superuser
```

Comment:
I could not find a single place where this context variable was being used for the help page or the TOS page